translate/src/modules/machinery/components/ConcordanceSearch.tsx

```diff
@@ -11,8 +11,8 @@ type Props = {
   translation: MachineryTranslation;
 };
 
-function ProjectList({ projects }: { projects: string[] }) {
-  const notEmpty = projects.filter(Boolean);
+function ProjectList({ projects }: { projects: (string | null)[] }) {
+  const notEmpty = projects.filter(Boolean) as string[];
```

Comment:
We should filter on something better here than asserting type, I think.

Reply:
To be fair, this is a bit on TS for not capturing the result of the filtering always being a `string[]`. As in, it doesn't matter what we `filter()` on, the typing result will still presume it could be null.